### PR TITLE
frontend/accountsummary: update data on syncdone event

### PR DESCRIPTION
**Pull request by benma**
This is the main event to listen to and is fired when an account is
updated.

### CODE REVIEW

File: frontends/web/src/routes/account/summary/accountssummary.tsx [hunk 1]
```diff
@@ -124,7 +124,11 @@ class AccountsSummary extends Component<Props, State> {
         if (data.type === 'account') {
             switch (data.data) {
                 case 'statusChanged':
+                case 'syncdone':
                     this.onStatusChanged(data.code);
+                    // Force getting account summary now; cancel nex scheduled call.
```
Comment (x1ddos) [inline frontends/web/src/routes/account/summary/accountssummary.tsx:129]: "next" typo

File: frontends/web/src/routes/account/summary/accountssummary.tsx [hunk 2]
```diff
@@ -95,7 +95,7 @@ class AccountsSummary extends Component<Props, State> {
     }
 
     public componentWillUnmount() {
-        window.clearInterval(this.summaryReqTimerID);
```
Comment (x1ddos) [inline frontends/web/src/routes/account/summary/accountssummary.tsx:98]: Good change.

> It's worth noting that the pool of IDs used by setTimeout() and setInterval() are shared, which means you can technically use clearTimeout() and clearInterval() interchangeably. However, for clarity, you should avoid doing so.

(from https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/clearTimeout)

Although "pool of IDs used by setTimeout() and setInterval() are shared" may depend on the browser, I believe.

**Review by x1ddos — APPROVED** [review]
LGTM but typo in "update data on syncodne event" commit title

**Comment by benma** [comment]
Omg my typo frequence is increasing rapidly :sob: 

Thanks. Fixed both.

**Pull request merged**